### PR TITLE
FBX-178 use context menu for timeline export

### PR DESCRIPTION
**Pull request by vkovec**

### CODE REVIEW

File: com.unity.formats.fbx/Editor/ExportModelEditorWindow.cs [hunk 1]
```diff
@@ -382,10 +383,19 @@ protected bool SelectionContainsPrefabInstanceWithAddedObjects()
         }
 
 
+        protected GameObject m_firstGameObjectToExport;
```
Comment (vkovec) [inline com.unity.formats.fbx/Editor/ExportModelEditorWindow.cs:386]: store the value as this gets called everytime the UI is updated

File: com.unity.formats.fbx/Tests/FbxTests/ExportTimelineClipTest.cs [hunk 2]
```diff
@@ -12,11 +12,8 @@ namespace FbxExporter.UnitTests
 {
     public class ExportTimelineClipTest : ExporterTestBase
     {
-#if !UNITY_2019_1_OR_NEWER
```
Comment (vkovec) [inline com.unity.formats.fbx/Tests/FbxTests/ExportTimelineClipTest.cs:15]: we don't support 2018 anymore

File: com.unity.formats.fbx/Editor/ConvertToPrefabEditorWindow.cs [hunk 3]
```diff
@@ -21,14 +21,14 @@ protected override bool DisableNameSelection
         {
             get
             {
-                return (GetToExport() != null && GetToExport().Length > 1);
+                return (ToExport != null && ToExport.Length > 1);
```
Comment (benoithudson) [inline com.unity.formats.fbx/Editor/ConvertToPrefabEditorWindow.cs:24]: Is this coming from a merge? It isn't bad exactly, just seems like noise.

But if we are changing things, here we could store ToExport in a local variable so we only call the getter once.

Comment (vkovec) [reply com.unity.formats.fbx/Editor/ConvertToPrefabEditorWindow.cs:24]: no, I just changed it from functions to a property

File: com.unity.formats.fbx/Tests/FbxTests/ExportTimelineClipTest.cs [hunk 4]
```diff
@@ -47,16 +44,52 @@ public void ExportSingleTimelineClipTest()
...
+                    Assert.That(atComponent, Is.Not.Null);
+
+                    var atObject = atComponent.gameObject;
+
+                    // One file by animation clip
+                    foreach (TimelineClip timeLineClip in at.GetClips())
+                    {
+                        Assert.That(timeLineClip.animationClip, Is.Not.Null);
+
+                        filePath = string.Format("{0}/{1}@{2}", folderPath, atObject.name, "Recorded.fbx");
```
Comment (benoithudson) [inline com.unity.formats.fbx/Tests/FbxTests/ExportTimelineClipTest.cs:80]: you can write this as `$"{folderPath}/{atObject.name}@Recorded.fbx"`

**Review by benoithudson — APPROVED** [review]

File: com.unity.formats.fbx/Editor/ConvertToPrefabEditorWindow.cs (hunk 3 above)
Comment (benoithudson) [reply com.unity.formats.fbx/Editor/ConvertToPrefabEditorWindow.cs]: Not a big deal but I'd recommend doing that in its own PR.

**Pull request merged**